the-quirky-web-dev/app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,3 @@
+@import "bootstrap";
+@import "./navbar/nav_bar";
```

Comment:
I needed to move this into a `navbar` folder [for compiling assets with sprockets](https://github.com/rails/sprockets/blob/master/UPGRADING.md#manifestjs)